huggingface/docker/Dockerfile_clm

```diff
@@ -0,0 +1,12 @@
+FROM mcr.microsoft.com/azureml/aifx/stable-ubuntu2004-cu115-py38-torch1110
+
+RUN python -m pip install \
+                        torch>=1.9.0 \
+                        datasets>=1.8.0 \
+                        sentencepiece>0.1.92 \
+                        transformers==4.16.0 \
+                        onnx>=1.9.0 \
+                        onnxruntime-training>=1.9.0 \
```

Comment:
I suggest using the latest release for ort as it will have all the new mem and perf optimizations.

Reply:
It was the suggested requirements when I added the run_clm (GPT2) script to Optimum repo -- but that's a good point. I'll try out the latest version of ORT and switch it to that!